src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs

```diff
@@ -66,6 +66,7 @@ private PlatformType GetPlatformType(DiagnosticSiteData postBody)
         {
             return postBody.IsXenon ?? false ? PlatformType.HyperV :
                 !string.IsNullOrWhiteSpace(postBody.Kind) && postBody.Kind.ToLower().Contains("kubernetes") ? PlatformType.Kubernetes :
+                !string.IsNullOrWhiteSpace(postBody.Kind) && postBody.Stamp.Kind.ToString().Contains("4") ? PlatformType.Kubernetes :
```

Comment:
You need to update this logic as well : (Line 44)

switch (app.Stamp.HostingEnvironmentType)
            {
                case HostingEnvironmentType.V1:
                    app.StampType = StampType.ASEV1;
                    break;
                case HostingEnvironmentType.V2:
                    app.StampType = StampType.ASEV2;
                    break;
                default:
                    app.StampType = StampType.Public;
                    break;
            }